mediapipe/examples/desktop/face_mesh_dll/canonical_mesh.h

```diff
@@ -0,0 +1,477 @@
+#include <vector>
+
+#include "mediapipe/framework/port/opencv_core_inc.h"
+
+namespace canonical {
+
+	std::vector<cv::Point3f> mesh{ cv::Point3f{0.000000,-3.406404,5.979507
```

Comment:
Please, use explicitly here initializer_list (additional "{" ). And fix the aligment of this file.
```suggestion
	std::vector<cv::Point3f> mesh{ { cv::Point3f{0.000000,-3.406404,5.979507
```